HPXMLtoOpenStudio/resources/hvac.rb

```diff
@@ -1804,6 +1808,252 @@ def self.disaggregate_fan_or_pump(model, fan_or_pump, htg_object, clg_object, ba
     end
   end
 
+  def self.apply_max_power_EMS(model, runner, hpxml_bldg, air_loop_unitary, control_zone, heating_system, htg_supp_coil = nil, clg_coil, htg_coil, schedules_file)
+    if not schedules_file.nil?
+      max_pow_ratio_sch = schedules_file.create_schedule_file(model, col_name: SchedulesFile::ColumnHVACMaximumPowerRatio, schedule_type_limits_name: Constants.ScheduleTypeLimitsFraction)
+      # Not allowed with unit multiplier for now
+      if not max_pow_ratio_sch.nil?
+        fail 'NumberofUnits greater than 1 is not supported for maximum power ratio schedules of variable speed hvac systems.' if hpxml_bldg.building_construction.number_of_units > 1
+      end
+    end
+    return if max_pow_ratio_sch.nil?
+    return if (clg_coil.nil? && htg_coil.nil?)
+
+    # Check maximum power ratio schedules only used in var speed systems,
+    clg_coil = nil unless clg_coil.is_a? OpenStudio::Model::CoilCoolingDXMultiSpeed
+    htg_coil = nil unless htg_coil.is_a? OpenStudio::Model::CoilHeatingDXMultiSpeed
+    htg_supp_coil = nil unless htg_coil.is_a? OpenStudio::Model::CoilHeatingDXMultiSpeed
+    # No variable speed coil
+    if (clg_coil.nil?) && (htg_coil.nil?)
+      runner.registerWarning('Maximum power ratio schedule is only supported for variable speed systems.')
+    end
+
+    if (htg_coil.is_a? OpenStudio::Model::CoilHeatingDXMultiSpeed) && (heating_system.backup_type != HPXML::HeatPumpBackupTypeIntegrated)
+      htg_coil = nil
+      htg_supp_coil = nil
+      runner.registerWarning('Maximum power ratio schedule is only supported for integrated backup system. Schedule is ignored for heating.')
+    end
```

Comment:
I suggest we remove this warning or make it more nuanced. It's common that we model MSHPs with backup electric baseboard and so we'll trigger this warning everytime. No one would expect the backup baseboard to use the power ratio schedule, so why produce a warning? It's more confusing than anything.